_docs_integrate/11-connector-configuration.md

```diff
@@ -280,8 +280,7 @@ interface WebhooksModulePayload {
 }
 ```
 
-The payload of the webhook is the same as the response payload of the `/api/v1/Account/Sync` endpoint. Thus the type `ConnectorSyncResult` of the [TypeScript SDK](./connector-sdks#typescript-sdk) can be used for specifing the webhook's payload type. Klick the links for a
-<a href="{% link _docs_explore/61-data-model.md %}#Message">Message Definition <i class="fas fa-fw fa-book"/></a> and
+Click the links for a <a href="{% link _docs_explore/61-data-model.md %}#Message">Message Definition <i class="fas fa-fw fa-book"/></a> and
```

Comment:
Wieso ist da ein Line Break zwischen "and" und dem Link zu der Relationship Definition?

Reply:
Dass die Zeile nicht so lang wird. Im Dokument ist dann kein break.